pkg/cmd/deploy.go

```diff
@@ -21,10 +21,9 @@ import (
 	"io"
 	"os/exec"
 
+	"github.com/ghodss/yaml"
```

Comment:
This constant changing order of imports is starting to get messy. 

We should do something with it.

Can we agree on having 3 blocks of imports, stdlib, gihtub.com/kedgeproject/kedge, and other?